Management/Clients/AirQualityDataClient.cs

```diff
@@ -98,12 +98,14 @@ public AirQualityDataClient(IConfiguration configuration)
             CountryCode countryCode,
             CancellationToken cancellationToken)
         {
-            var sampledCities = await GetDefaultCitiesAsync(state, countryCode, cancellationToken);
```

Comment:
Why are you changing the logic here?

**It is calling the api to get list of supported cities**, now after your change it's calling the fixed data.

Reply:
Hope you can see my comments now. I have restored the logic back :)